src/components/Main/utils.js

```diff
@@ -0,0 +1,21 @@
+import {
+  FISCAL_YEAR_ROUTE,
+  FUNDS_ROUTE,
+  GROUPS_ROUTE,
+  LEDGERS_ROUTE,
+} from '../../common/const';
+
+const viewPermissionsMap = {
+  'ui-finance.ledger.view': LEDGERS_ROUTE,
+  'ui-finance.fiscal-year.view': FISCAL_YEAR_ROUTE,
+  'ui-finance.group.view': GROUPS_ROUTE,
+  'ui-finance.fund-budget.view': FUNDS_ROUTE,
+};
+
+export const getInitialRoute = (stripes) => {
+  const routesList = Object.keys(viewPermissionsMap)
+    .map(perm => (stripes.hasPerm(perm) ? viewPermissionsMap[perm] : ''))
```

Comment:
what is empty string mean here? It's not correct semantically to use empty string like this.

also the whole function looks over-complicated. I would make it in more clean way,
something like 
routePermissionMap;
enabledRoutes = Object.keys(routePermissionMap).filter(route => stripes.hasPerm(routePermissionMap[route]))